### PR TITLE
Model/species delete

### PR DESCRIPTION
**Pull request by Ollekmc**
Trying to be more vertical 😃

### CODE REVIEW

File: frontend/src/component/SpeciesApp.tsx [hunk 1]
```diff
@@ -1,15 +1,15 @@
-import useSpecies from "../hooks/useSpecies";
+import useSpeciesPlural from "../hooks/useSpeciesPlural";
 import AddSpecies from "./AddSpecies";
 import SpeciesGallery from "./SpeciesGallery";
 
 export default function SpeciesApp(){
-    const {speciesPlural, addSpecies} = useSpecies()
+    const {speciesPlural, addSpecies, removeSpecies} = useSpeciesPlural()
```
Comment (NickKlock) [inline frontend/src/component/SpeciesApp.tsx:6]: useSpeciesPlural() funny one :D

**Review by NickKlock — APPROVED** [review]
LGTM ! nice work

File: frontend/src/hooks/useSpeciesPlural.ts [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    function getSpeciesPlural(){
+        axios.get("api/plants")
+            .then(response => {
+                setSpeciesPlural(response.data)
+            })
+            .catch(console.error)
+    }
+
+    function addSpecies(newSpecies: NewSpecies) {
+        console.log(newSpecies)
```
Comment (Flooooooooooorian) [inline frontend/src/hooks/useSpeciesPlural.ts:22]: remove console.log

File: frontend/src/hooks/useSpeciesPlural.ts [hunk 3]
```diff
@@ -0,0 +1,39 @@
+import {NewSpecies, Species} from "../model/Species";
+import axios from "axios";
+import {useEffect, useState} from "react";
+
+export default  function useSpeciesPlural() {
```
Comment (Flooooooooooorian) [inline frontend/src/hooks/useSpeciesPlural.ts:5]: Im not really happy with the naming. "useSpeciesPlural" maybe i would name it "useSpecies" and "useSingleSpecies" or "useDetailedSpecies"

**Review by Flooooooooooorian — APPROVED** [review]
LGTM

**Pull request merged**